### PR TITLE
TileGroup to Tailwind

### PR DESCRIPTION
**Pull request by oreqizer**
https://kiwicom.atlassian.net/browse/FEPLT-1744
 Storybook: https://orbit-mainframev-oreqizer-tw-tile-group.surge.sh

### CODE REVIEW

**Review by mainframev — CHANGES_REQUESTED** [review]
LGTM 👍🏻 Just a few small things

File: packages/orbit-components/src/TileGroup/index.tsx [hunk 1]
```diff
@@ -1,57 +1,25 @@
 "use client";
 
 import * as React from "react";
-import styled, { css } from "styled-components";
+import clsx from "clsx";
```
[the remark this answers is not in the document]
Comment (oreqizer) [reply packages/orbit-components/src/TileGroup/index.tsx:4]: adjusted 👍

File: packages/orbit-components/src/TileGroup/index.tsx [hunk 2]
```diff
@@ -1,57 +1,25 @@
...
-};
-
 const TileGroup = ({ children, dataTest, id, as }: Props) => {
+  const Component = (as ?? "div") as React.ElementType;
+
   return (
-    // @ts-expect-error FIXME: ts-migrate
-    <StyledTileGroup data-test={dataTest} as={as} id={id}>
+    <Component
+      className={clsx(
```
[the remark this answers is not in the document]
Comment (oreqizer) [reply packages/orbit-components/src/TileGroup/index.tsx:13]: adjusted 👍

File: packages/orbit-components/src/TileGroup/index.tsx [hunk 3]
```diff
@@ -1,57 +1,25 @@
...
+        "shadow-action rounded-normal [&_.orbit-slide]:bg-white-normal m-0 w-full p-0",
+        "[&_.orbit-tile-wrapper]:first:rounded-t-normal [&_.orbit-tile-wrapper]:last:rounded-b-normal [&_.orbit-tile-wrapper]:not-last:border-b [&_.orbit-tile-wrapper]:not-last:border-cloud-normal",
+        "[&_.orbit-tile-wrapper]:duration-fast [&_.orbit-tile-wrapper]:hover:bg-white-normal [&_.orbit-tile-wrapper]:focus:bg-white-normal [&_.orbit-tile-wrapper]:rounded-none [&_.orbit-tile-wrapper]:shadow-none [&_.orbit-tile-wrapper]:transition-all [&_.orbit-tile-wrapper]:ease-in-out",
```
[the remark this answers is not in the document]
Comment (oreqizer) [reply packages/orbit-components/src/TileGroup/index.tsx:16]: adjusted 👍

**Review by mainframev — APPROVED** [review]

File: packages/orbit-components/src/TileGroup/index.tsx [hunk 4]
```diff
@@ -1,57 +1,25 @@
...
-    }
-  `}
-`;
-
-StyledTileGroup.defaultProps = {
-  theme: defaultTheme,
-};
-
 const TileGroup = ({ children, dataTest, id, as }: Props) => {
+  const Component = (as ?? "div") as React.ElementType;
```
Comment (DSil) [inline packages/orbit-components/src/TileGroup/index.tsx:9]: This `as` prop was actually not doing anything, was it? It seems to always just render a `div` 🤔

File: packages/orbit-components/src/TileGroup/index.tsx [hunk 5]
```diff
@@ -1,57 +1,25 @@
...
+
   return (
-    // @ts-expect-error FIXME: ts-migrate
-    <StyledTileGroup data-test={dataTest} as={as} id={id}>
+    <Component
+      className={cx(
+        "shadow-action rounded-normal [&_.orbit-slide]:bg-white-normal m-0 w-full p-0",
+        "[&_.orbit-tile-wrapper]:first:rounded-t-normal [&_.orbit-tile-wrapper]:last:rounded-b-normal [&_.orbit-tile-wrapper]:not-last:border-b [&_.orbit-tile-wrapper]:not-last:border-cloud-normal",
```
Comment (DSil) [inline packages/orbit-components/src/TileGroup/index.tsx:15]: These selectors are wrong. The `first:`, `last:` and `not-last:` selectors refer to the `.orbit-tile-wrapper`, therefore, must come first

```suggestion
        "first:[&_.orbit-tile-wrapper]:rounded-t-normal last:[&_.orbit-tile-wrapper]:rounded-b-normal not-last:[&_.orbit-tile-wrapper]:border-b not-last:[&_.orbit-tile-wrapper]:border-cloud-normal",
```

File: packages/orbit-components/src/TileGroup/index.tsx [hunk 6]
```diff
@@ -1,57 +1,25 @@
...
+        "[&_.orbit-tile-wrapper]:duration-fast [&_.orbit-tile-wrapper]:hover:bg-white-normal [&_.orbit-tile-wrapper]:focus:bg-white-normal [&_.orbit-tile-wrapper]:rounded-none [&_.orbit-tile-wrapper]:shadow-none [&_.orbit-tile-wrapper]:transition-colors [&_.orbit-tile-wrapper]:ease-in-out",
```
Comment (DSil) [inline packages/orbit-components/src/TileGroup/index.tsx:16]: These selectors are wrong. The hover: and focus: selectors refer to the `.orbit-tile-wrapper`, therefore, must come first.
Also, the bg colors are wrong.

```suggestion
        "[&_.orbit-tile-wrapper]:duration-fast hover:[&_.orbit-tile-wrapper]:bg-cloud-normal focus:[&_.orbit-tile-wrapper]:bg-cloud-normal [&_.orbit-tile-wrapper]:rounded-none [&_.orbit-tile-wrapper]:shadow-none [&_.orbit-tile-wrapper]:transition-colors [&_.orbit-tile-wrapper]:ease-in-out",
```

**Pull request merged**